source/mentorprogrammeapplication.md

```diff
@@ -43,7 +43,7 @@ hope to gain from it and the mentor programme
  * A brief CV of your skills
 
 Note, we do not expect people to come to our mentor programme with a full
-armoury of skills, however mentors are not their to teach you only to guide
+armoury of skills, however mentors are not here to teach you, but only to guide
```

Comment:
Disclaimer: non-native English here. Updating `their` to `there` makes most sense IMO. Any input here @cottage14 ?

```suggestion
armoury of skills, however mentors are not there to teach you, but only to guide
```